src/components/ContentHome/styled.ts

```diff
@@ -0,0 +1,93 @@
+import styled from 'styled-components'
+import czechtech20191 from '../../assets/Home/czechtech2019-1.jpg'
+
+export const Wrapper = styled.article`
+	display: grid;
+	grid-template-columns: 1fr 6fr 1fr;
+	margin: 1em;
+	justify-items: center;
+
+	@media (max-width: 50em) {
+		margin: 0;
+		padding-top: 3em;
+		padding-bottom: 1em;
+	}
+`
+
+export const HomeSection = styled.section`
+	grid-column: 2;
+	display: flex;
+	flex-flow: row nowrap;
+	align-items: center;
+	gap: 2em;
+	max-width: 60em;
```

Comment:
I think the max-width property is unnecessary. Or I would increase the width. 60em is very little.

Reply:
Now I'm looking at global styles and seeing that the home page has been fixed 👍